ui/src/main/resources/index.html

```diff
@@ -52,5 +52,6 @@
       <!-- }}} -->
     </div>
     <script type="text/javascript" src="bundle.js"></script>
+    <script type="text/javascript" src="/admin/iframe-resizer/2.6.2/js/iframeResizer.contentWindow.min.js"></script>
```

Comment:
I think for now we are going to leave the reference to the admin-ui script because it simplifies our build process and we can't match the exact version otherwise.